packages/block-library/src/post-excerpt/edit.js

```diff
@@ -168,17 +193,19 @@ export default function PostExcerptEditor( {
 			value={
 				isSelected
 					? rawOrRenderedExcerpt
-					: ( trimmedExcerpt !== '...' ? trimmedExcerpt : '' ) ||
+					: ( ! isTrimmed
+							? rawOrRenderedExcerpt
+							: trimmedExcerpt + '…' ) ||
```

Comment:
To make it obvious at a glance that this is ellipsis, how about make it a variable, similar to the approach [here](https://github.com/WordPress/gutenberg/blob/83088ab864addc27e60df5a2bfbd9a61d97ea717/packages/components/src/truncate/utils.ts#L7)?